cmd/network/packet-capture.go

```diff
@@ -517,15 +517,19 @@ func waitForPacketCapturePod(o *packetCaptureOptions, capturePod *corev1.Pod) er
 }
 
 func setCaptureInterface(o *packetCaptureOptions) error {
-	ds := &appsv1.DaemonSet{}
-	if err := o.kubeCli.Get(context.TODO(), types.NamespacedName{Namespace: "openshift-ovn-kubernetes", Name: "ovnkube-master"}, ds); err != nil && !apierrors.IsNotFound(err) {
+	networkConfig := &configv1.Network{}
+	if err := o.kubeCli.Get(context.Background(), client.ObjectKey{Name: "cluster"}, networkConfig); err != nil {
 		return fmt.Errorf("failed to determine the network type: %s", err)
 	}
 
-	if ds.Status.DesiredNumberScheduled > 0 {
+	switch networkConfig.Spec.NetworkType {
+	case "OVNKubernetes":
 		o.captureInterface = "genev_sys_6081"
 		return nil
+	case "OpenShiftSDN":
+		o.captureInterface = "vxlan_sys_4789"
+		return nil
+	default:
+		return fmt.Errorf("failed to determine network type or unknown network type")
```

Comment:
One last `nit`: let's put the value in this string so it's quick to spot if it comes up :)